src/demo/demo.test.js

```diff
@@ -0,0 +1,46 @@
+import { h } from 'preact'
+import { mount } from 'enzyme'
+import { expect } from 'chai'
+
+describe('Mount Demo App', () => {
+  let Demo = null
+
+  beforeEach(() => (Demo = require('./demo.js')))
+
+  describe('by mocking Onfido SDK', () => {
+    // Mock window.Onfido
+    global.Onfido = {
+      init: () => {},
+    }
+
+    it('mounts the Onfido Demo without crashing', () => {
+      // the component needs to be assigned to a lowercase variable to work!
+      const sdk = <Demo />
+      const sdkDemo = mount(<sdk />)
+      expect(sdkDemo).to.be.ok
```

Comment:
The Chai documentation doesn't recommend using this assertion chain pattern as it is not a strict comparison https://www.chaijs.com/api/bdd/#method_ok
Perhaps what you want is this
```suggestion
      expect(sdkDemo.exists()).toBe(true)
```

Reply:
Actually it should be `toEqual()` instead of `toBe()`